logstash-core/lib/logstash/runner.rb

```diff
@@ -394,4 +353,7 @@ def trap_sigint
     end
   end
 
-end # class LogStash::Runner
+  def setting(key)
+    @settings.get_value(key)
+  end
+end
```

Comment:
I expect we see this pattern used in a lot of places.
I which we had some kind of module or base classes to define theses useful methods.